.licenses/npm/@actions/artifact.dep.yml

```diff
@@ -1,6 +1,6 @@
 ---
 name: "@actions/artifact"
-version: 0.6.1
+version: 1.0.0
```

Comment:
Just curious, why the bump from `0.6.1` to `1.0.0` here?

Reply:
So rationale here was that versions < `1.0.0` are "pre-release" versions.  Since `@actions/artifact` is released and used in production we wanted to give it a `v1` version number. `v1.0.0` is backwards compatible with previous versions.

Reply:
Good catch with `@actions/checkout@v3` 😄 ! Didn't know that existed. I think upgrading `checkout` to use `checkout@v3` should be done in a separate PR.